site/docs/v0.14.3/README.md

```diff
@@ -135,26 +135,26 @@ welcome pull requests. Feel free to dig through the [issues][issue] and jump in.
 
 See [the list of releases][releases] to find out about feature changes.
 
-[airgap]: docs/airgap.md
+[airgap]: airgap.md
 [brew]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos
 [cncf]: https://github.com/cncf/k8s-conformance#certified-kubernetes
 [coc]: https://github.com/heptio/sonobuoy/blob/master/CODE_OF_CONDUCT.md
 [contrib]: https://github.com/heptio/sonobuoy/blob/master/CONTRIBUTING.md
-[conformance]: docs/conformance-testing.md
+[conformance]: conformance-testing.md
 [docs]: https://github.com/heptio/sonobuoy/tree/master/docs
-[e2e]: /docs/conformance-testing.md
-[gen]: docs/gen.md
+[e2e]: conformance-testing.md
+[gen]: gen.md
 [gimme]: https://github.com/travis-ci/gimme
 [issue]: https://github.com/heptio/sonobuoy/issues
 [k8s]: https://github.com/kubernetes/kubernetes
 [linux]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1
 [oview]: https://youtu.be/k-P4hXdruRs?t=9m27s
```

Comment:
We should get a newer recording/demo up to better explain the project and demos of new features.
Not blocking the merge of course.